packages/planner/src/app/components_ngrx/iterations-panel/iterations-panel.component.html

```diff
@@ -3,9 +3,10 @@
   <!-- Active iteration -->
   <div class="f8-itr-hdr nohover">
     <div class="f8-itr__add" [class.show]="sidePanelOpen" [class.hide]="!sidePanelOpen">
-      <a
-        *ngIf="loggedIn && editEnabled"
-        class="text-right pointer"
+      <button
+        [disabled]="editEnabled"
+        class="text-right pointer f8-itr__btn"
+        [ngClass]="{'f8-itr__disable' : editEnabled}"
```

Comment:
You can disable ```a``` tag without changing it to ```button``` tag. So, need to add ```[disable]``` just add these properties to ```f8-itr__disable``` class.
```pointer-events: none;```, ```color: #9b9d9d;``` and ```cursor: not-allowed;```